helm_deploy/values-preprod.yaml

```diff
@@ -10,8 +10,10 @@ generic-service:
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.dev.json
     HMPPS_AUTH_URL: https://sign-in-preprod.hmpps.service.justice.gov.uk/auth
     PRISON-API_BASE-URL: https://api-preprod.prison.service.justice.gov.uk
-    SENTRY_ENVIRONMENT: preprod
     PRISONER-SEARCH-API_BASE-URL: https://prisoner-search-preprod.prison.service.justice.gov.uk
+    PRISON-REGISTER-API_BASE-URL: https://prison-register-preprod.hmpps.service.justice.gov.uk
+
```

Comment:
MINOR: do we want a blank line here? If yes, then we might need changing in outer envs as well